spec/support/config/capybara.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rspec/rails'
+require 'capybara/rspec'
+require 'capybara-screenshot/rspec'
+require 'selenium-webdriver'
+require 'site_prism'
+require 'site_prism/all_there'
+require_relative 'support/pages/home'
```

Comment:
Remove it

Reply:
https://github.com/site-prism/site_prism#using-siteprism-with-cucumber

Reply:
> Remove it

![image](https://user-images.githubusercontent.com/11984862/114295418-cfb99d80-9aad-11eb-882c-5d5324970352.png)